.github/workflows/ci.yml

```diff
@@ -81,49 +81,49 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        ember-try-scenario: [
-          ember-lts-3.20,
-          ember-lts-3.24,
-          ember-lts-3.28,
-          ember-release,
-          ember-beta,
-          ember-default-with-jquery,
-          ember-classic,
-          embroider-safe,
-          embroider-optimized
-        ]
+        ember-try-scenario:
+          [
+            ember-lts-3.28,
+            ember-lts-4.4,
+            ember-release,
+            ember-beta,
+            ember-default-with-jquery,
```

Comment:
I think we can remove this scenario as well? At least, it's not part of the default blueprint anymore.